src/main/zapHomeFiles/hud/tools/activeScan.js

```diff
@@ -40,7 +40,7 @@ var ActiveScan = (function() {
 		registerForZapEvents(ACTIVE_SCAN_EVENT);
 	}
 
-	function showDialog(tabId, domain) {
+	function showStdDialog(tabId, domain) {
```

Comment:
Sorry to leave once last nit pick. Was there a reasn to change this function name? Across almost all of the other tools we use just `showDialog` would be nice to leave it the same if possibe.

Reply:
Yeah, its because I introduced a new dialog for scanning just this request but then later removed it.
Will change back to the old one :)

Reply:
Done